<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>759</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Floatworld</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="GridWidget" name="gridWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Grid</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuOccupant">
    <property name="title">
     <string>Occupant</string>
    </property>
    <addaction name="actionNextOccupant"/>
    <addaction name="actionPrevOccupant"/>
    <addaction name="actionIndividualStep"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOccupant"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionStop"/>
   <addaction name="actionPlay1_4"/>
   <addaction name="actionPlay1_2"/>
   <addaction name="actionPlay1"/>
   <addaction name="actionPlay2"/>
   <addaction name="actionPlay4"/>
   <addaction name="actionActionFF"/>
   <addaction name="separator"/>
   <addaction name="actionStep"/>
   <addaction name="actionIndividualStep"/>
   <addaction name="actionPrevOccupant"/>
   <addaction name="actionNextOccupant"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockGridSettings">
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>100</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>World</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="gridBox">
    <zorder>gridWidget</zorder>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockInspector">
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>100</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="occupantBox"/>
  </widget>
  <action name="actionPlay1">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/play-2.png</normaloff>:/icons/play-2.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Ordinary speed</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/red-stop.png</normaloff>:/icons/red-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/right-stop.png</normaloff>:/icons/right-stop.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/document-new.svg</normaloff>:/icons/document-new.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/document-save.svg</normaloff>:/icons/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/document-open.svg</normaloff>:/icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionPlay2">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/play-3.png</normaloff>:/icons/play-3.png</iconset>
   </property>
   <property name="text">
    <string>Fast</string>
   </property>
   <property name="toolTip">
    <string>2x speed</string>
   </property>
  </action>
  <action name="actionIndividualStep">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/right-stop-1.png</normaloff>:/icons/right-stop-1.png</iconset>
   </property>
   <property name="text">
    <string>Step Selected</string>
   </property>
   <property name="toolTip">
    <string>Individual Step</string>
   </property>
   <property name="shortcut">
    <string>/</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNextOccupant">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/green-right.png</normaloff>:/icons/green-right.png</iconset>
   </property>
   <property name="text">
    <string>Next occupant</string>
   </property>
   <property name="shortcut">
    <string>&gt;</string>
   </property>
  </action>
  <action name="actionPrevOccupant">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/green-left.png</normaloff>:/icons/green-left.png</iconset>
   </property>
   <property name="text">
    <string>Prev occupant</string>
   </property>
   <property name="toolTip">
    <string>Previous occupant</string>
   </property>
   <property name="shortcut">
    <string>&lt;</string>
   </property>
  </action>
  <action name="actionPlay1_2">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/play-1.png</normaloff>:/icons/play-1.png</iconset>
   </property>
   <property name="text">
    <string>Slow</string>
   </property>
   <property name="toolTip">
    <string>1/4 speed</string>
   </property>
  </action>
  <action name="actionPlay1_4">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/play-0.png</normaloff>:/icons/play-0.png</iconset>
   </property>
   <property name="text">
    <string>Slowest</string>
   </property>
   <property name="toolTip">
    <string>1/20 speed</string>
   </property>
  </action>
  <action name="actionPlay4">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/play-4.png</normaloff>:/icons/play-4.png</iconset>
   </property>
   <property name="text">
    <string>Fastest</string>
   </property>
   <property name="toolTip">
    <string>8x speed</string>
   </property>
  </action>
  <action name="actionActionFF">
   <property name="icon">
    <iconset resource="floatworld.qrc">
     <normaloff>:/icons/blue-right-right.png</normaloff>:/icons/blue-right-right.png</iconset>
   </property>
   <property name="text">
    <string>actionFF</string>
   </property>
   <property name="toolTip">
    <string>Fast-forward</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GridWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="floatworld.qrc"/>
 </resources>
 <connections/>
</ui>
